server/odc-common/src/main/java/com/oceanbase/odc/common/util/StringUtils.java

```diff
@@ -396,4 +398,37 @@ public static String getBriefSql(String sql, Integer maxLength) {
         }
     }
 
+    public static boolean startsWithIgnoreSpaceAndNewLines(@NonNull String target, @NonNull String prefix,
+            boolean ignoreCase, int maxMatchLength) {
+        char[] targets;
+        char[] prefixes;
+        if (ignoreCase) {
+            targets = target.toLowerCase().toCharArray();
+            prefixes = prefix.toLowerCase().toCharArray();
+        } else {
+            targets = target.toCharArray();
+            prefixes = prefix.toCharArray();
+        }
+        int matchLength = 0;
+        int j = 0;
+        for (int i = 0; i < targets.length && j < prefixes.length && matchLength < maxMatchLength;) {
+            char iChar = targets[i];
+            char jChar = prefixes[j];
+            if (iChar == ' ' || iChar == '\t' || iChar == '\n' || iChar == '\r') {
```

Comment:
if other white space should be included? refer java.lang.Character.isWhitespace()

Reply:
done